backend/uclapi/oauth/scoping.py

```diff
@@ -19,6 +19,8 @@ class Scopes:
     SCOPE_MAP = {
         "timetable": (1, "Personal Timetable"),
         "student_number": (2, "Student Number"),
+        "libcal_read": (3, " Read LibCal Bookings"),
+        "libcal_write": (4, " Write LibCal Bookings"),
```

Comment:
@Feverfew I feel two scopes is a better idea. I think most apps are just going to read someone's personal booking rather than also try and make one. Reduces the risk of a single developer suddenly deciding they would like to remove everyone's bookings.